app/src/main/java/com/sdp/movemeet/chat/ChatActivity.java

```diff
@@ -186,43 +205,6 @@ public void logout(View view) {
 
     }
 
-    private void settingUpMessageAdapter() {
```

Comment:
What is the purpose of creating a class  for your custom FirebaseRecyclerAdapter instead of defining it in this method ? Do you intend to use the adapter in other classes as well ? Or is it just to have less lines of codes in this class ?

Reply:
No, it is not intended to be used elsewhere. You're right, the purpose was here lighten the code in ChatActivity and to make the distinction with the MessageAdapter even clearer!